### ISSUE TITLE
[CHORE] 5.0.0 release

### ISSUE THREAD

**Issue by aclist** [aclist/kbin-kes#269]
- [ ] Fix concat funcs script to clear ESLint errors (dangling commas)
- [ ] Abstract fetch subscriptions function to shared utility functions
- [ ] Use fetch subs function in checkmarks mod
- [ ]  Abstract getComputedFontSize() function to safeGM
- [ ] Implement stricter CSS on noti panel (cf. #251)
- [ ] Expand troubleshooting section in docs with common issues
    - https://developer.chrome.com/docs/extensions/reference/api/userScripts#developer_mode_for_extension_users
- [ ] Add screenshots to links field
    - [ ] Document purpose of link/preview key in manifest

**Comment by melroy89** [comment]
You might want to rename it to mbin/kes

**Comment by aclist** [comment]
It seems reasonable. The name can be changed entirely if necessary. But I have not thought of a satisfactory alternative yet.

**Comment by melroy89** [comment]
Yeah, I'm afraid /kbin is no more. Also feel free to contribute directly to the Mbin core. So your improvements will be available for everybody by default, without the need for additional CSS scripts. Assuming there are some CSS improvements that is beneficial for everybody. And even then, special features can be placed behind settings/switches. 


We are on Matrix: [https://matrix.to/#/#mbin-general:melroy.org](https://matrix.to/#/#mbin-general:melroy.org).

![image](https://github.com/user-attachments/assets/ba83f2c0-70a9-459e-b604-83b89ac2117d)

**Comment by aclist** [comment]
This ticket is being closed and subdivided into atomic tickets.
